src/locale/languages.ts

```diff
@@ -47,8 +48,9 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.ru, name: 'Русский – Russian'},
   {code2: AppLanguage.tr, name: 'Türkçe – Turkish'},
   {code2: AppLanguage.uk, name: 'Українська – Ukrainian'},
-  {code2: AppLanguage.zh_CN, name: '简体中文（中国）– Chinese (Simplified)'},
-  {code2: AppLanguage.zh_TW, name: '繁體中文（臺灣）– Chinese (Traditional)'},
+  {code2: AppLanguage.zh_CN, name: '简体中文 – Simplified Chinese',},
```

Comment:
```suggestion
  {code2: AppLanguage.zh_CN, name: '简体中文 – Simplified Chinese'},
```
There's an extra `,` here which I think may be causing a lint error :)

Reply:
Thanks!